internal/commands/app/create_test.go

```diff
@@ -605,32 +604,17 @@ Check out your app: cd ./remote-app && realm-cli app describe
 		console.Tty().Close() // flush the writers
 		<-doneCh              // wait for procedure to complete
 
-		appLocal, err := local.LoadApp(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name, local.BackendPath))
+		_, err = local.LoadApp(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name, local.BackendPath))
 		assert.Nil(t, err)
 
 		backendFileInfo, err := ioutil.ReadDir(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name, local.BackendPath))
 		assert.Nil(t, err)
-		assert.Equal(t, len(backendFileInfo), 9)
+		assert.Equal(t, len(backendFileInfo), 6)
 
 		frontendFileInfo, err := ioutil.ReadDir(filepath.Join(profile.WorkingDirectory, cmd.inputs.Name, local.FrontendPath, templateID))
 		assert.Nil(t, err)
 		assert.Equal(t, len(frontendFileInfo), 1)
 		assert.Equal(t, frontendFileInfo[0].Name(), "react-native")
-
```

Comment:
I think this is now somewhat flaky because I added a spinner for template app creation.

I removed these assertions because they are unnecessary for testing whether "should create a new app from a template" is true.

Reply:
i'd be curious to see if adding that `Log` post-spinner de-flakes these asserts, but otherwise 👍